src/data/enat/basic.lean

```diff
@@ -34,7 +34,8 @@ instance : is_well_order ℕ∞ (<) := { }
 
 variables {m n : ℕ∞}
 
-@[simp, norm_cast] lemma coe_zero : ((0 : ℕ) : ℕ∞) = 0 := rfl
+-- eligible for `dsimp`
+@[simp, nolint simp_nf, norm_cast] lemma coe_zero : ((0 : ℕ) : ℕ∞) = 0 := rfl
```

Comment:
What changed that made `simp_nf` start complaining?

Reply:
`char_p.cast_eq_zero` was marked `simp`